packages/plugins/src/plugins/renzo/types.ts

```diff
@@ -0,0 +1,16 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+
+export type RenzoContractConfig = {
+  address: `0x${string}`;
```

Comment:
There's a stronger typing available

```suggestion
  address: Address;
```